.ci/integration/gnupg-git/default.yml

```diff
@@ -132,6 +142,15 @@
       command: bash -lc "cd {{ gpg_src_path }} && ./configure {{ gpg_build_config }}"
       changed_when: False
 
+    # gnupg on alpine is not detecting absence of fig2dev when this thread suggests it should: https://lists.gnupg.org/pipermail/gcrypt-devel/2013-December/002710.html
+    - name: Disable making docs on Alpine
```

Comment:
What is the consequence of not making docs on Alpine? 
[It seems like we still generate man pages for the apk?]

Reply:
Good question that needs clarification in the comments. These steps hit github and pull the GPG code and builds that which wants ImageMagik to conversion of SVG to PND and other graphical stuff that git-secret doesn't use it it's man page docs. The fact that we don't generate the graphics for GPG docs has no effect on all our git-secret tests, packaging and docs.